google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -441,6 +460,13 @@ private Value() {}
    */
   public abstract String getString();
 
+  /**
+   * Returns the value of a {@code JSON}-typed instance.
+   *
+   * @throws IllegalStateException if {@code isNull()} or the value is not of the expected type
+   */
+  public abstract String getJson();
```

Comment:
```suggestion
  public String getJson() {
    throw new UnsupportedOperationException("Not implemented");
  }
```

Reply:
We should provide a default implementation not to break any users who have their own implementation of the interface.